File: 509 Fibonacci Number.cs
public class Solution {
    public int Fib(int n) {
        int first = 0;
        int second = 1;
        while (n-- > 0) {
            int tmp = first;
            first = second;
            second = tmp + second;
        }
        return first;
    }
}

//  04/16/2021 11:19	Accepted	32 ms	15.2 MB	csharp

File: 509 Fibonacci Number.java
class Solution {
    public int fib(int n) {
        if (n < 2) {
            return n;
        }
        int a = 0; 
        int b = 1; 
        int temp;
        
        for (int i = 1; i < n; i++) {
            temp = a;
            a = b;
            b += temp;
        }
        return b;
    }
}

// 04/16/2021 11:28	Accepted	0 ms	35.4 MB	java
